streams-bootstrap-core/src/main/java/com/bakdata/kafka/util/TopicClient.java

```diff
@@ -55,6 +57,12 @@
 @Slf4j
 public final class TopicClient implements AutoCloseable {
 
+    private static final RetryConfig RETRY_CONFIG = RetryConfig.<Boolean>custom()
+            .retryOnResult(result -> result)
+            .failAfterMaxAttempts(false)
+            .maxAttempts(3)
+            .waitDuration(Duration.ofSeconds(5L))
```

Comment:
Would it make sense to make all or some of these settings configurable?

Reply:
I don't see a need for this atm. If it is later needed, we can add it